lib/dialogs/logout-confirmation/index.tsx

```diff
@@ -32,56 +34,45 @@ export class LogoutConfirmation extends Component<Props> {
 
     return (
       <div className="logoutConfirmation">
-        <Dialog onDone={closeDialog} title="Unsynchronized Notes Found">
-          <section>
-            <h1>Are you sure you want to logout?</h1>
-          </section>
-
+        <Dialog onDone={closeDialog} title="Unsynced Notes">
+          <p className="explanation">
+            {notes.size > 0
+              ? 'Logging out will delete any unsynced notes.'
+              : 'All notes have syncronized!'}
+          </p>
+
+          <p className="explanation-secondary">
+            {notes.size > 0 && 'Possibly unsynchronized notes'}
+          </p>
           {notes.size > 0 && (
-            <p className="explanation">
-              If you logout now Simplenote will not be able to remember any of
-              your unsynchronized changes. If you wait a few more seconds it
-              might catch up with the changes but if it doesn&apos;t you can
-              always export a copy of the notes with those changes just to be
-              safe.
-            </p>
+            <section className="change-list">
+              <ul>
+                {[...notes.entries()].map(([noteId, note]) => {
+                  const { title, preview } = noteTitleAndPreview(note);
+
+                  return (
+                    <li key={noteId}>
+                      <AttentionIcon />
```

Comment:
when I saw the screenshot with the new icon I wondered if it might not be fun to keep the notes in the list but instead to replace the icon with one confirming the sync status.

that is, instead of removing the indicator that it could be unsynchronized…_add_ an indicator that it _has_ been synchronized

Reply:
The designs did not have a resolved state. I do like this idea!!

Reply:
I just realized that this will increase the complexity in the code since we grab this data in a stateless fashion.

Should just need some initial state when the component mounts to store the initial set of note ids and then on render we can compare that set against the set passed in to know if they are confirmed or not.

Reply:
Im thinking we leave that for another PR.